### DISCUSSION TITLE
I can't add a GPG key and the error doesn't say what the problem is

### DISCUSSION THREAD

**Discussion by OrsellGit** [community/community#38030] [New to GitHub] [answered]
I can't seem to add a new GPG key to my account, this is the first one I've ever done, so I've followed the main guide and everything. But when I submit it to the GPG prompt in my settings, I get this error: 
![image](https://user-images.githubusercontent.com/34631691/199380428-ecd02b18-f017-4594-af9b-66cc3ab0c87a.png)
I can commit and sign with the GPG key, but I can't add it to my account, so I can't have my commits marked as Verified. It doesn't bother to give me a pop-up or any other message about the error. I've even made a new GPG key just in case it didn't like my first one.

Any suggestions?
Thanks

**Comment by ossanna16** [comment]
Hi there @OrsellGaming and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by OrsellGit** [comment, marked as answer]
The solution was that I forgot to include the "-----BEGIN PGP PUBLIC KEY BLOCK-----" and "-----END PGP PUBLIC KEY BLOCK-----" of the public key export block. The guide mentions them but it should be a little more specific.
